tests/testthat/test-backend-mssql.R

```diff
@@ -160,3 +160,12 @@ FROM `df`")
     sql("SELECT `x`, CASE\nWHEN (((`x`) IS NULL)) THEN (1.0)\nWHEN (NOT(((`x`) IS NULL))) THEN (2.0)\nELSE (3.0)\nEND AS `z`\nFROM `df`")
   )
 })
+
+test_that("ORDER BY in subqueries uses TOP 100 PERCENT (#175)", {
+  mf <- lazy_frame(x = 1:3, con = simulate_mssql())
+
+  expect_equal(
```

Comment:
Maybe it's better to use `expect_known_output()` here?

Reply:
This code is consistent with the rest of the file. I filed #285 to address this.